src/public/dashboard.html

```diff
@@ -2,10 +2,22 @@
 <html>
 <head>
   <title>Say What!</title>
-  <link href="views/css/dashboard.css" rel="stylesheet" type="text/css" />
+  <link href="dashboard.css" rel="stylesheet" type="text/css" />
```

Comment:
pretty sure we should keep the css here

Reply:
I fixed the css bro. Just merge my changes.

Reply:
K sure

Reply:
And it works? Like you tested it out

Reply:
yes. merge them changes and all the css and js words.